bridges/snowbridge/pallets/system/src/lib.rs

```diff
@@ -663,6 +726,42 @@ pub mod pallet {
 			let secondary_exists = Channels::<T>::contains_key(SECONDARY_GOVERNANCE_CHANNEL);
 			primary_exists && secondary_exists
 		}
+
+		pub(crate) fn do_register_token(
+			location: &Location,
+			metadata: AssetMetadata,
+			pays_fee: PaysFee<T>,
+		) -> Result<(), DispatchError> {
+			let ethereum_location = Location::new(2, [GlobalConsensus(T::EthereumNetwork::get())]);
```

Comment:
nit:
You could get the full (relative) location of Ethereum from the Runtime, rather than getting only `NetworkId` and hardcoding the relative location in the pallet.

```suggestion
			let ethereum_location = T::EthereumLocation::get();
```